test/test_socks.py

```diff
@@ -263,15 +263,15 @@ def ctx(request):
 
 
 class TestSocks4Proxy:
-    @pytest.mark.parametrize('handler,ctx', [('Urllib', 'http')], indirect=True)
+    @pytest.mark.parametrize('handler,ctx', [('Urllib', 'http'), ('Requests', 'http')], indirect=True)
```

Comment:
You can also parametrize the whole class to achieve the same.

I have it separate currently because some of the handlers may not support some tests, and in the case of the socks ones they may have different protocols to test it on.

It's definitely one of the things I'd like to clean up when we get around to rewriting/cleaning up the networking tests when requirements are more clear. Currently it's just making sure there is at least some sort of test.